### ISSUE TITLE
[BUG] - Log spammed with missing jelly.png file

### ISSUE THREAD

**Issue by StephenShamakian** [b3rs3rk/gpustat-unraid#35]
**Describe the bug**
Unraid Log is being spammed with missing jelly.png file every time the dashboard page is hit with my browser.

**To Reproduce**
1. Refresh Dashboard page
2. Look at Unraid Log
3. Look at Browser Dev Tools (error 404 for jelly.png)

**Expected behavior**
I think jelly.png was renamed to jellyfin.png?
As i see a jellyfin.png file in "/usr/local/emhttp/plugins/gpustat/images/" but not file named "jelly.png"

**Screenshots**
![image](https://user-images.githubusercontent.com/6382844/155059720-4f0f8365-a17e-49e4-beb4-4860a7844f07.png)

**Client (please complete the following information):**
 - OS: Windows 10
 - Browser Chrome
 - Version 98.0.4758.102

**Server (please complete the following information):**

 - OS Version: [e.g. UnRAID v6.10.0-rc2
 - GPU: 2x Nvidia P2000

**Comment by b3rs3rk** [comment]
@StephenShamakian should be fixed in this latest release -- please test and let me know.  Thanks!

**Comment by StephenShamakian** [comment]
@b3rs3rk Yes, I can confirm the latest version has fixed this issue! Thanks!
